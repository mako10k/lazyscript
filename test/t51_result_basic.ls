!{
  ~R <- ~~include "lib/Result.ls";
  ~~println (~to_str (~R.Result.Ok 1));
  ~~println (~to_str (~R.map (\~x -> ~~add ~x 1) (~R.Result.Ok 2)));
  ~~println (~to_str (~R.withDefault 9 (~R.Result.Err "e")));
};
