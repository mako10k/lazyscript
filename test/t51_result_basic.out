Ok 1
Ok 3
9
()
